Source/Core/Core/PowerPC/Jit64/Jit_LoadStore.cpp

```diff
@@ -264,6 +335,14 @@ void Jit64::dcbx(UGeckoInstruction inst)
   BT(32, R(value), R(addr));
   FixupBranch invalidate_needed = J_CC(CC_C, true);
 
+  if (make_loop)
+  {
+    ADD(32, R(effective_address), Imm8(32));
+    MOV(32, R(value), R(effective_address));
+    SUB(32, R(loop_counter), Imm8(1));
```

Comment:
Would it be possible/advantageous to use `DEC` here?

Reply:
https://github.com/dolphin-emu/dolphin/blob/ffdc8538a162b1ca56e3943c5a78c02358053b2a/Source/Core/Common/x64Emitter.h#L409-L413

Though who knows how accurate that even is anymore, that comment is from 2008. But either way it's not in our Jit so...

Reply:
Let's leave it as it is then.